### ISSUE TITLE
Detect out of date manifest on startup

### ISSUE THREAD

**Issue by lucacasonato** [denoland/fresh#123]
One of the major pitfalls of fresh is that the manifest needs to be updated after adding islands or routes by using `fresh manifest`. If you forget to do this, pages just don't show up. While it is not possible for us to automatically update the manifest at runtime, we can detect discrepancies between the disk and the manifest at startup and just error out until the user runs `fresh manifest`.

Let's do that.

**Comment by tylerbrostrom** [comment]
What are your thoughts re: a new `--watch` option for `fresh manifest`?

If you think it’s worth exploration, do you suppose it’d be dependent or complementary to the stated issue? (i.e. Could they be worked on in parallel?)

(BTW, I really admire your’s and others’ work on this project—especially the JIT bundler. Truly liberating DX.)

**Comment by lucacasonato** [comment]
Thanks @tylerbrostrom! Great to hear you are liking it :-)

**Comment by NfNitLoop** [comment]
I just set my deno.json to:

```json
{
  "tasks": {
    "start": "fresh manifest && deno run -A --watch --no-check main.ts"
  }
}
```

I do have to manually restart if I add a new file, but I'm used to that from some other frameworks.

**Comment by lucacasonato** [comment]
This will essentially be resolved by #223
